mozfun/hist/merge/metadata.yaml

```diff
@@ -0,0 +1,4 @@
+description: Merge an array of histograms into a single histogram. - The histogram
+  values will be summed per-bucket - The count will be summed - Other fields will
+  take the mode_last
```

Comment:
We'll probably need to manually change whitespace here to ensure the list renders well.